plugin.lua

```diff
@@ -7,19 +7,19 @@
 
 -- allow for require to search relative to this plugin file
 -- open for improvements!
+
+---Notes: confirm "path/to/lua-language-server/script/", in Lua.Workspace.Library
+---Notes: confirm "path/to/bee" in Lua.Workspace.Library
+
 if not __plugin_dev and not _G.__factorio_plugin_initialized then
   _G.__factorio_plugin_initialized = true
 
-  ---@type table
```

Comment:
Inferred correctly when "path/to/lua-language-server/script/", is in Lua.Workspace.Library